### ISSUE TITLE
Build now failing w/ requirements.txt that does a pip install from GitHub repo

### ISSUE THREAD

**Issue by jonathansick** [rtfd/readthedocs.org#1718]
There is a new build issue related to installing dependencies from a `requirements.txt` file that in turn installs a repo from GitHub. (https://github.com/lsst/LDM-129/blob/master/requirements.txt)

This worked last week (September 23) but is now failing without any changes on my part. I wonder if this is related to the [new containerized infrastructure](https://blog.readthedocs.com/securing-build-processes/)?

Specifically, this is my build error log:

```
Requirement already satisfied (use --upgrade to upgrade): PyYAML>=3.11 in /home/docs/checkouts/readthedocs.org/user_builds/ldm-129/envs/tickets-dm-3836/lib/python2.7/site-packages (from -r /home/docs/checkouts/readthedocs.org/user_builds/ldm-129/checkouts/tickets-dm-3836/requirements.txt (line 1))
Requirement already satisfied (use --upgrade to upgrade): Sphinx>=1.3.1 in /home/docs/checkouts/readthedocs.org/user_builds/ldm-129/envs/tickets-dm-3836/lib/python2.7/site-packages (from -r /home/docs/checkouts/readthedocs.org/user_builds/ldm-129/checkouts/tickets-dm-3836/requirements.txt (line 2))
Obtaining lsst-dd-rtd-theme from git+git://github.com/lsst-sqre/lsst_dd_rtd_theme.git@master#egg=lsst_dd_rtd_theme (from -r /home/docs/checkouts/readthedocs.org/user_builds/ldm-129/checkouts/tickets-dm-3836/requirements.txt (line 3))
  Cloning git://github.com/lsst-sqre/lsst_dd_rtd_theme.git (to master) to /home/docs/checkouts/readthedocs.org/user_builds/ldm-129/envs/tickets-dm-3836/src/lsst-dd-rtd-theme
fatal: unable to connect to github.com:
github.com[0: 192.30.252.130]: errno=Connection timed out

  Complete output from command git clone -q git://github.com/lsst-sqre/lsst_dd_rtd_theme.git /home/docs/checkouts/readthedocs.org/user_builds/ldm-129/envs/tickets-dm-3836/src/lsst-dd-rtd-theme:
  
  ----------------------------------------
Command "git clone -q git://github.com/lsst-sqre/lsst_dd_rtd_theme.git /home/docs/checkouts/readthedocs.org/user_builds/ldm-129/envs/tickets-dm-3836/src/lsst-dd-rtd-theme" failed with error code 128 in None
```

In summary, a ``git clone -q git://github.com/lsst-sqre/lsst_dd_rtd_theme.git`` is timing out because it can't reach GitHub's servers. The command works normally outside the RTD build container.

**Comment by agjohnson** [comment]
I'm guessing this is a firewalling issue between the build containers and the git protocol port. We'll get a fix in on the port today

**Comment by jonathansick** [comment]
Thanks @agjohnson!

**Comment by agjohnson** [comment]
I pushed out a change to allow access to the port, which should address this issue.

**Comment by jonathansick** [comment]
It works! Much thanks for the quick fix.

**Comment by anthokalel** [comment]
How did you do that @agjohnson  ?
